docs/dev_101_services.md

```diff
@@ -138,6 +138,12 @@ set_speed:
 The name and description of the service actions are set in our [translations](/docs/internationalization/core#services) and not in the service action description. Each service action and service action field must have a matching translation defined.
 :::
 
+### Grouping of service action fields
+
+As shown in the example above, input fields can be visually grouped in sections. Grouping input fields by sections influence
+only how the inputs are displayed to the user, and not how service action data is structured. In the example above, service
+action data would be `{"speed_pct": 50}`, not `{"advanced_fields": {"speed_pct": 50}}`.
```

Comment:
```suggestion
only how the inputs are displayed to the user, and not how service action data is structured.
In the example above, service action data would be `{"speed_pct": 50}`, not `{"advanced_fields": {"speed_pct": 50}}`.
```
I don't recall right off the bat if we have rules on how the lines should be wrapped, but I personally like it when a complete sentence ends the line as this makes the diffs nicer to read and review.